src/components/NavBar.js

```diff
@@ -17,18 +19,29 @@ function NavBar() {
         </Link>
 
       <div>
-        <button
-          type="button"
-          onClick={() => history.push('/signin')}
-        >
-          Log in
-        </button>
-        <button
-          type="button"
-          onClick={() => history.push('/signup')}
-        >
-          Registreren
-        </button>
+        {isAuth === false ?
+            <>
+            <button
+                type="button"
+                onClick={() => navigate('/signin')}
+            >
+              Log in
+            </button>
+            <button
+                type="button"
+                onClick={() => navigate('/signup')}
+            >
+              Registreren
+            </button>
+            </>
+            :
+            <button
+                type="button"
+                onClick={logoutFunction}
+            >
+              Uitloggen
+            </button>
+        }
```

Comment:
Probeer wanneer je een ternary operator gebruikt om de boolean expressie makkelijk leesbaar te houden. Nu valt `isAuth === false` wel mee (en had je dus eigenlijk `isAuth` willen hebben). Je gaat in je programmeercarrière vast nog wel een langere expressie maken met meer variabelen, en zodra die meer `=== false` gaan krijgen met nog `||` en/of `&&` operators, dan is het moeilijk te lezen